rpc/legacyrpc/methods.go

```diff
@@ -324,8 +268,14 @@ func jsonError(err error) *dcrjson.RPCError {
 
 // accountAddressIndex returns the next address index for the passed
 // account and branch.
-func accountAddressIndex(icmd interface{}, w *wallet.Wallet) (interface{}, error) {
+func accountAddressIndex(s *Server, icmd interface{}) (interface{}, error) {
 	cmd := icmd.(*dcrjson.AccountAddressIndexCmd)
+	w, ok := s.walletLoader.LoadedWallet()
+
```

Comment:
please remove these unneeded newlines between the load and the check, there were several introduced by this pr.